ui.c

```diff
@@ -51,8 +51,9 @@ int setupUI() {
   post_menu(song_menu);
   wrefresh(song_win);
 
-  while ((ch = getch()) != KEY_F(1)) {
-    mvprintw(1, COLS - 7, "%d", getSongTime());
+  while ((ch = getch()) != 'q') {
```

Comment:
```suggestion
  while ('q' != (ch = getch()) ) {
```
does this also apply?